stable/postgresql/templates/deployment.yaml

```diff
@@ -62,7 +62,11 @@ spec:
         - name: POSTGRES_PASSWORD
           valueFrom:
             secretKeyRef:
+            {{- if .Values.existingSecret }}
+              name: {{ .Values.existingSecret }}
```

Comment:
how about `name: {{ default ( include "postgresql.fullname" ) .Values.existingSecret |  quote }}`?
I'm curious if that would be easier to read?

Reply:
I definitely think the if version is easier to read. It took me a few seconds to understand the deafult version 😃 but may just not be enough experience writting helm charts. 

I do prefer the if statement more because semantically we are not defaulting in a missing value, it's more like 2 different code paths based on whether helm is managing the secret vs the user. 

That being said I could change it to either way.